app/src/main/java/com/kamron/pogoiv/activities/OcrCalibrationResultActivity.java

```diff
@@ -105,7 +105,11 @@ public static void startCalibration(@NonNull Context context,
             sCalibrationImage = bitmap;
         } else {
             // Make a mutable copy of the bitmap so we can draw on it with a Canvas
-            sCalibrationImage = bitmap.copy(sCalibrationImage.getConfig() ,true);
```

Comment:
Please replace this line with this instead:

`sCalibrationImage = bitmap.copy(bitmap.getConfig(), true);`

Reply:
That's also what I thought originally but I checked the reference and realized getConfig() could actually return NULL.
```
If the bitmap's internal config is in one of the public formats, return that config, otherwise return null.
```
Maybe I just make an if there